setup.py

```diff
@@ -33,7 +33,7 @@ def do_setup(ext_modules):
     setup(
         name="Orange3-Associate",
         description="Orange add-on for mining frequent itemsets and association rules.",
-        long_description=open(path.join(path.dirname(__file__), 'README.rst')).read(),
+        long_description=open(path.join(path.dirname(__file__), 'README.pypi')).read(),
```

Comment:
If not using [setup.cfg](https://setuptools.readthedocs.io/en/latest/setuptools.html#configuring-setup-using-setup-cfg-files) with content type inferred from file extension, this is probably missing [`long_description_content_type='text/markdown'`](https://packaging.python.org/guides/making-a-pypi-friendly-readme/#including-your-readme-in-your-package-s-metadata).